roles/lib_openshift_api/library/oc_secrets.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/env python
+'''
+module for openshift cloud secrets
+'''
+#   Examples:
+#
+#  # to initiate and use /etc/origin/master/admin.kubeconfig file for auth
+#  - name: list secrets
+#    oca_secrets:
```

Comment:
Removed oca.  oca is what we use for our admin command. oc + admin.  I have updated it to be oc in the examples.